sandbox/src/mocks/r4/searchServiceRequest/responses/ResponseExampleAdvice.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "123e4567-e89b-12d3-a456-426614174002",
+    "resourceType": "Bundle",
+    "meta": {
+        "profile": [
+            "https://fhir.hl7.org.uk/StructureDefinition/UKCore-Bundle"
+        ],
+        "lastUpdated": "2022-03-01T10:00:00.42Z"
+    },
+    "timestamp": "2022-04-06T15:42:00.42Z",
+    "link": [
+        {
+            "relation": "self",
+            "url": "_baseUrl_/ServiceRequest?identifier=https://fhir.nhs.uk/Id/UBRN%7C000000070001"
```

Comment:
I think this should have R4 in the path as the base path ends /FHIR

Reply:
Good spot, will include R4 in all occurrences.